lib/order.rb

```diff
@@ -0,0 +1,34 @@
+class Order
+attr_accessor :lines, :total
+
+  def initialize
+    @lines = []
+    @total = 0
+  end
+
+  def add(line)
+    lines.push(line)
+  end
+
+  def totalNoTax
+    lineTotals = self.lines.map {|line| line[1]* line[2]}
+    lineTotals.inject(:+)
+  end
+
+  def totalWithTax
+    (totalNoTax * (100 + TAX)/ 100).round(2)
+  end
+
+  def finish
+    @total = totalWithTax
+  end
+
+  def print
+    self.lines.each do |line|
+      puts "#{line[0]} - #{line[1]} x #{line[2]}\n"
+    end
+    puts "Tax: #{(self.totalNoTax * 8.64/100).round(2)}\n"
```

Comment:
Surrounding space missing for operator '/'.<br>Do not write to stdout. Use Rails' logger if you want to log.